app/schemas/FOOD.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "foodId": {
+      "type": "string"
+    },
+    "name": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "nutritionalData": {
```

Comment:
I think we should also have the following fields. They can be empty strings for now, but i think having it in the schema would be a good idea:

- gluten_free: boolean
- dairy_free: boolean
- vegetarian: boolean
- vegan: boolean
- keto: boolean (maybe, this one seems less important)